### DISCUSSION TITLE
[Bug]: File integrity checks fails after update to NC 26.0.2 (AIO) and some apps

### DISCUSSION THREAD

**Discussion by afischer211** [nextcloud/all-in-one#2737] [Questions] [answered]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

I updated my AIO-installation of Nextcloud to the most current version (AIO 6.0.0 and NC 26.0.2). Shortly after this I also updated some of the installed apps like Calendar.
Now I have a failure on the file integrity check with following affected files:
- core
	- INVALID_HASH
		- core/js/mimetypelist.js
	- EXTRA_FILE
		- core/img/filetypes/off.svg
		- core/img/filetypes/3dm.svg
		- core/img/filetypes/bim.svg
		- core/img/filetypes/fcstd.svg
		- core/img/filetypes/fbx.svg
		- core/img/filetypes/iges.svg
		- core/img/filetypes/brep.svg
		- core/img/filetypes/glb.svg
		- core/img/filetypes/ifc.svg
		- core/img/filetypes/wrl.svg
		- core/img/filetypes/dae.svg
		- core/img/filetypes/3ds.svg
		- core/img/filetypes/3mf.svg
		- core/img/filetypes/step.svg
		- core/img/filetypes/mtl.svg
		- core/img/filetypes/stl.svg
		- core/img/filetypes/obj.svg
		- core/img/filetypes/ply.svg
		- core/img/filetypes/gltf.svg

### Steps to reproduce

1. Install NC AIO 6.0.0 and NC 26.0.2
2. ???

### Expected behavior

No integrity problems (or a clean way to solve this)

### Installation method

Official All-in-One appliance

### Nextcloud Server version

26

### Operating system

Other

### PHP engine version

Other

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

Updated from a MINOR version (ex. 22.1 to 22.2)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by szaimen** [comment, marked as answer]
> * INVALID_HASH
>   
>   * core/js/mimetypelist.js

Hi, this is nothing to worry about: you can just ignore it.

It usually should get fixed automatically with the next Nextcloud update which is planned for beginning of July.

As for the other problems, they are likely caused by one of your installed apps and should be fixed automatically when disabling the problematic app and also waiting for the next Nextcloud update.
